src/test/java/com/commercetools/sync/services/impl/CustomObjectServiceImplTest.java

```diff
@@ -263,17 +266,17 @@ void fetchMatchingCustomObjects_WithAllEmptyKey_ShouldNotFetch() {
         when(client.execute(any())).thenReturn(CompletableFuture.completedFuture(result));
 
         final Set<CustomObject<JsonNode>> customObjects = service
-                .fetchMatchingCustomObjects(customObjectCompositeIdentifiers)
-                .toCompletableFuture().join();
+            .fetchMatchingCustomObjects(customObjectCompositeIdentifiers)
+            .toCompletableFuture().join();
 
```

Comment:
this indentation looks weird

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/569/commits/a8c7ce4c770a85be1efb2f658ea00f9ee8c58393